spock-specs/src/test/groovy/org/spockframework/smoke/condition/SetComparisonRendering.groovy

```diff
@@ -0,0 +1,111 @@
+package org.spockframework.smoke.condition
+
+class SetComparisonRendering extends ConditionRenderingSpec {
+
+  def "Set differences are displayed"() {
+    expect:
+    isRendered('''
+a == b
```

Comment:
would be nicer with `.stripMargin()`. Probably another marker should be used instead of pipe.